addons/prometheus-operator/kube-green/resources/servicemonitor.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     scrapeTimeout: 30s
     metricRelabelings:
     - action: keep
-      regex: controller_runtime.*
+      regex: (controller_runtime|kube_green_).*
```

Comment:
`controller_runtime` metrics are needed even with the custom ones?

Reply:
Yes, it's an addition of the default metrics

Reply:
Update the CHANGELOG